### DISCUSSION TITLE
Video import JPG vs PNG

### DISCUSSION THREAD

**Discussion by osinpl** [cvat-ai/cvat#4102] [Ideas]
Hey! why imported video is converted into PNG images, not eg JPG, is there any limitation or advantage?

Happy New Year!

br, Kris

**Comment by nmanovic** [comment]
@osinpl , the primary idea is to keep the original quality (png is lossless). In CVAT we have two types of data: original and compressed. First one is used for exporting datasets, second one is used for annotations.

**Reply by osinpl** [reply]
> @osinpl , the primary idea is to keep the original quality (png is lossless). In CVAT we have two types of data: original and compressed. First one is used for exporting datasets, second one is used for annotations.

Hi, thx for fast reply! Yes I understand PNG idea, and you are right it is losses, anyway most of the data ppl use for annotation are already compressed, so exporting it in PNG will not make any difference. Btw most of the videos are compressed (mp4). The problem is that PNG is quite heavy and when we want to speed up training, jpg format could help. Can you let me know which module is responsible for choosing this format when we do export? Thank you in advance! You made me curious and will look for publications that compare PNG vs JPG used model training! Happy New Year!

**Comment by nmanovic** [comment]
@osinpl , you can find corresponding code here: https://github.com/openvinotoolkit/cvat/blob/f915385f2c7c10b7004aee55d524f37514658c43/cvat/apps/engine/frame_provider.py#L47

**Reply by osinpl** [reply]
Thank you!
